.craft.yml

```diff
@@ -0,0 +1,12 @@
+minVersion: "0.15.0"
+github:
+  owner: getsentry
+  repo: snuba-sdk
+releaseBranchPrefix: releases
+changelog: CHANGES
+changelogPolicy: auto
+artifactProvider:
+  name: github
+targets:
+  - name: github
+  - name: pypi
```

Comment:
You may want to use `requiredNames` to make sure you have _some_ artifacts, like we do over at Sentry: https://github.com/getsentry/sentry/blob/1a539eaf3fed64a7085b80166be90bc6d20b1f14/.craft.yml#L38-L39